Bento/Renderable/ViewStorage.swift

```diff
@@ -0,0 +1,50 @@
+public struct ViewStorage {
+    internal let componentType: Any.Type
+    internal let view: BentoReusableView
+
+    internal init(componentType: Any.Type, view: BentoReusableView) {
+        self.componentType = componentType
+        self.view = view
+    }
+
+    public subscript<Value>(key: Key<Value>) -> Value? {
+        get {
+            let key = StorageKey(component: componentType, identifier: key.identifier)
+            return view.storage[key] as! Value?
```

Comment:
Why not `as? Value`? Is there any difference?

Reply:
`as! Value?` asserts that in presence of the value, it would always be `Value`. Since the key is unique, and is tied to a particular type, this is a precondition that should be held.